neuralmonkey/decorators.py

```diff
@@ -0,0 +1,26 @@
+#tests: lint
+from functools import wraps
+from neuralmonkey.logging import debug
+
+def placeholder(func):
+    @wraps(func)
```

Comment:
wraps dělá, že když to žuchne zevnitř tadytý vnitřní dekorátorský funkce, nebo když tam budeš používat věci jako __module__ nebo tak, tak se to bude přejímat z toho modulu, odkud ten dekorátor voláš a bude to ukazovat tam, kam chceš místo sem do tohohle dekorátoru.